packages/@o3r/configuration/collection.json

```diff
@@ -6,6 +6,11 @@
       "factory": "./schematics/ng-add/index#ngAdd",
       "schema": "./schematics/ng-add/schema.json",
       "aliases": ["install", "i"]
+    },
+    "add-config": {
```

Comment:
To simplify the usage, the `ng add` of `@o3r/configuration` should add itself into the angular.json's `schematicCollections`.
You can use the `registerPackageCollectionSchematics` rule from `@o3r/schematics`